doc/code/qml_fermi.rst

```diff
@@ -0,0 +1,92 @@
+qml.fermi
+=========
+
+Overview
+--------
+
+This module contains functions and classes for creating and manipulating fermionic operators.
+
+.. currentmodule:: pennylane.fermi
+
+.. automodapi:: pennylane.fermi
+    :no-heading:
+    :no-main-docstr:
+    :no-inherited-members:
+
+FermiC and FermiA
+-----------------
+
+The fermionic creation and annihilation operators are constructed with :class:`~pennylane.FermiC`
+and :class:`~pennylane.FermiA`. We pass in the index of the orbital that the
+operator acts on. For example, the operators :math:`a^{\dagger}_0` and :math:`a_3`, acting on the
+:math:`0`th and :math:`3`rd orbitals, are constructed as
```

Comment:
```suggestion
:math:`0`\th and :math:`3`\rd orbitals, are constructed as
```

I _think_ this will fix the failing documentation thing. Otherwise:

```suggestion
:math:`0\text{th}` and :math:`3\text{rd}` orbitals, are constructed as
```